### PR TITLE
feat: add yaml file with current version, to easier fetch images corr…

### PR DESCRIPTION
**Pull request by aepfli**
…elating with gherking file

closes: https://github.com/open-feature/flagd-testbed/issues/152

### CODE REVIEW

File: version.yaml
Comment (beeme1mr) [inline version.yaml:1]: I think this should be a `version.txt`

File: release-please-config.json [hunk 1]
```diff
@@ -5,7 +5,10 @@
       "release-type": "simple",
       "prerelease": false,
       "bump-minor-pre-major": true,
-      "bump-patch-for-minor-pre-major": true
+      "bump-patch-for-minor-pre-major": true,
+      "extra-files": [
+        "version.yaml"
+      ]
     }
```
Comment (beeme1mr) [inline release-please-config.json:12]: This shouldn't be required if you have a `version.txt` file.

File: version.yaml [hunk 2]
```diff
@@ -0,0 +1 @@
+version: v0.5.17
```
Comment (beeme1mr) [inline version.yaml:1]:
```suggestion
0.5.17
```

Comment (aepfli) [reply version.yaml]: can i leave the v in front of it? or is this a hard nogo?

Comment (aepfli) [reply version.yaml]: nevermind, not as important, that is something i can assume too

Comment (beeme1mr) [reply version.yaml]: I'm not sure. You could try it, and we could verify it in the release PR.

**Review by beeme1mr — APPROVED** [review]

**Pull request merged**